tox.ini

```diff
@@ -0,0 +1,66 @@
+[tox]
+minversion = 1.6
+envlist =
+  docs,
+  jjb,
+  jjb-compare-xml,
+  jjb-version,
+  license,
+  pre-commit,
+skipsdist = True
+
+[testenv:docs]
+basepython = python3
+deps = -rrequirements.txt
+commands =
+    sphinx-build -W -b html -n -d {envtmpdir}/doctrees ./docs/ {toxinidir}/docs/_build/html
+
+[testenv:jjb]
+basepython = python3
+deps =
+    jenkins-job-builder==3.5.0
+commands =
+    jenkins-jobs -l DEBUG test --recursive -o {toxinidir}/archives/job-configs {toxinidir}/jjb:{toxinidir}/.jjb-test
+
+[testenv:jjb-compare-xml]
+basepython = python3
+deps =
+    jenkins-job-builder
+commands =
+    ./jjb-compare-xml.sh
```

Comment:
This tox job is being defined to call a script that isn't in the tree. Have you run tox locally on this?

Reply:
jjb-version and jjb will fail for now since I havent brough in the jjb folder.